opendevin/server/listen.py

```diff
@@ -222,9 +222,13 @@ def list_files(request: Request, path: str = '/'):
             status_code=status.HTTP_404_NOT_FOUND,
             content={'error': 'Runtime not yet initialized'},
         )
-
     try:
-        return request.state.session.agent_session.runtime.file_store.list(path)
+        files: list[str] = request.state.session.agent_session.runtime.file_store.list(
```

Comment:
After selecting a sub_dir, file_store points to the sub_directory.
Then after a refresh, it lists the sub_dir's next-level folders. Should there be a separate end-point and a file_store to fetch the workspace_base contents